modules/generator/registry/registry.go

```diff
@@ -87,6 +87,8 @@ type metric interface {
 	removeStaleSeries(staleTimeMs int64)
 }
 
+const HighestAggregationInterval = 1 * time.Minute
```

Comment:
We don't need to make this value public
```suggestion
const highestAggregationInterval = 1 * time.Minute
```
(you'll need to update all references too)